pkg/operator/ceph/cluster/controller.go

```diff
@@ -174,6 +174,10 @@ func (c *ClusterController) onK8sNodeAdd(obj interface{}) {
 			logger.Debugf("Skipping -> Do not use all Nodes")
 			continue
 		}
+		if cluster.Info == nil {
```

Comment:
I believe `cluster.Info.IsInitialized()` would have been more appropriate.

Reply:
@leseb for this case we'd need to ensure `cluster.Info` was either not a pointer (my preference) or make sure its always allocated and avoid nil as a sentinel value?